aiida_fleur/tests/workflows/test_banddos_workchain.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+# Copyright (c), Forschungszentrum Jülich GmbH, IAS-1/PGI-1, Germany.         #
+#                All rights reserved.                                         #
+# This file is part of the AiiDA-FLEUR package.                               #
+#                                                                             #
+# The code is hosted on GitHub at https://github.com/JuDFTteam/aiida-fleur    #
+# For further information on the license, see the LICENSE.txt file            #
+# For further information please visit http://www.flapw.de or                 #
+# http://aiida-fleur.readthedocs.io/en/develop/                               #
+###############################################################################
+
+# Here we test if the interfaces of the workflows are still the same
+from __future__ import absolute_import
+from __future__ import print_function
+
+import pytest
+import aiida_fleur
+import os
+
+aiida_path = os.path.dirname(aiida_fleur.__file__)
+TEST_INP_XML_PATH = os.path.join(aiida_path, 'tests/files/inpxml/Si/inp.xml')
+CALC_ENTRY_POINT = 'fleur.fleur'
+
+
+# tests
+@pytest.mark.usefixtures("aiida_profile", "clear_database")
```

Comment:
Could the explicit fixtures use be removed? I think both of them will be autoused